sunpy/map/tests/test_mapbase.py

```diff
@@ -1041,6 +1046,35 @@ def test_superpixel_masked(aia171_test_map_with_mask):
     assert superpix_map.dimensions[1] == expected_shape[1] - 1 * u.pix
 
 
+def test_superpixel_masked_conservative_mask_warning(aia171_test_map_with_mask):
```

Comment:
Add another unit test to verify that both `np.sum`+`conservative_mask=True` and `np.mean`+`conservative_mask=False` do _not_ emit a warning